vue/components/ui/molecules/animated-button-icon/animated-button-icon.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="animated-button-icon">
+        <button-icon
+            class="button button-icon-front"
+            v-bind:icon="icon"
+            v-bind:color="color"
+            v-bind:size="size"
+            v-bind:disabled="disabled"
+            v-bind:class="{ visible: !clicked }"
+            v-on:click="onClick"
+        />
+        <button-icon
+            class="button button-icon-click"
+            v-bind:icon="clickIcon"
+            v-bind:color="color"
+            v-bind:size="size"
+            v-bind:disabled="true"

```

Comment:
Are you sure we want this? I think it changes the UI, and we don't want that. Maybe disabling pointer events is enough.